api-analytics/server.js

```diff
@@ -9,17 +9,18 @@ var swig = require('swig');
 // This is the collection of logs.
 var requestsLog = [];
 
-// And these are the default quotations. `makeDefaults()` simply add the id and the
+// And these are the default quotations.
```

Comment:
hah, I just did a soft reset to the last master commit, and started staging interactively :)